beacon-chain/p2p/monitoring.go

```diff
@@ -63,6 +64,80 @@ var (
 		Name: "p2p_sync_committee_subnet_attempted_broadcasts",
 		Help: "The number of sync committee that were attempted to be broadcast.",
 	})
+
+	// Gossip Tracer Metrics
+	pubsubTopicsActive = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "p2p_pubsub_topic_active",
+		Help: "The topics that the peer is participating in gossipsub.",
+	},
+		[]string{"topic"})
+	pubsubTopicsGraft = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "p2p_pubsub_graft_count",
+		Help: "The number of graft messages sent for a particular topic",
+	},
+		[]string{"topic"})
```

Comment:
Should this be `p2p_pubsub_graft_total`?
Should this be a counter (only increases) instead of a gauge?

https://prometheus.io/docs/practices/naming/

Reply:
Same feedback for all of these metrics